jerry-core/include/jerryscript-port.h

```diff
@@ -111,27 +111,19 @@ void JERRY_ATTR_FORMAT (printf, 2, 3) jerry_port_log (jerry_log_level_t level, c
  */
 
 /**
- * Jerry time zone structure
- */
-typedef struct
-{
-  int offset;                /**< minutes from west */
-  int daylight_saving_time;  /**< daylight saving time (1 - DST applies, 0 - not on DST) */
-} jerry_time_zone_t;
-
-/**
- * Get timezone and daylight saving data
+ * Get local timezone adjustment, in milliseconds. This value, added to a timestamp in UTC
+ * time, should yield local time. Ideally, this function should satisfy the stipulations
+ * applied to LocalTZA in section 20.3.1.7 of ecmascript version 9.0 [1].
+ * [1] https://www.ecma-international.org/ecma-262/9.0/#sec-local-time-zone-adjustment
```

Comment:
IIRC, the code base doesn't use url references to external documents, it just mentions the standard name, the version, and the section. Like:
```c
  * See also:
  *          ECMA-262 v5, 15.9.1.9
```
(Taken from another code block, needs to be adapted.)